### ISSUE TITLE
When creating new user, NOT NULL constraint failed: Users.MaxActiveSessions

### ISSUE THREAD

**Issue by jayache80** [jellyfin/jellyfin#5046]
**Describe the bug**
In the Admin dashboard -> Server -> Users, when I try to create a new user, an error toast appears:
"There was an error processing the request. Please try again later"

The log file shows a `Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 19: 'NOT NULL constraint failed: Users.MaxActiveSessions'` exception.

**System (please complete the following information):**
 - OS: Arch linux
 - Virtualization: N/A
 - Clients: Browser
 - Browser: Firefox 84
 - Jellyfin Version: 10.6.4
 - Playback: N/A
 - Installed Plugins: N/A
 - Reverse Proxy: N/A
 - Base URL: /Users/New
 - Networking: LAN
 - Storage: NFS

**To Reproduce**

1. Go to Admin Dashboard
2. Under "Server" click on "Users"
3. Click "+" to add new user
4. Fill out form
5. Submit form and see error message.

**Expected behavior**
No error message should appear, and a new user should've been added.

**Logs**
```
[2021-01-17 21:49:02.594 -08:00] [ERR] [114] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: "http://192.168.1.5:8096/Users/New"
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 19: 'NOT NULL constraint failed: Users.MaxActiveSessions'.
   at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
   at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Jellyfin.Server.Implementations.Users.UserManager.CreateUser(String name)
   at MediaBrowser.Api.UserService.Post(CreateUserByName request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

**Screenshots**
![jellyfin-add-user-error](https://user-images.githubusercontent.com/15721514/104881216-c63cf100-5915-11eb-83e6-4b2fa949b163.png)

**Additional context**
N/A

**Comment by jayache80** [comment]
There was already an issue created for this: #4922 and was resolved by updating.
I had problems with 10.7 RC1 but I'll try again with 10.7 RC2 to ensure this bug is indeed no longer an issue.

**Comment by jayache80** [comment]
Indeed, this issue is fixed in 10.7 RC2 (and perhaps already fixed in RC1, but I didn't test that).
